### PR TITLE
badblocks: add more examples

### PR DESCRIPTION
**Pull request by dhatch48**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/badblocks.md [hunk 1]
```diff
@@ -14,3 +14,11 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to file:
+
+`sudo badblocks -o {{badblocks.txt}} -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode with improved speed using 4K block size and 64K block count:
```
Comment (zdroid) [inline pages/common/badblocks.md]: 4K and 64K correspond to 4096 and 65536 right under? If that's the case, then those numbers should be put in inside braces such as `{{4096}}`. Perhaps `with improved speed` is unnecessary.

File: pages/common/badblocks.md [hunk 2]
```diff
@@ -14,3 +14,11 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to file:
+
+`sudo badblocks -o {{badblocks.txt}} -w {{/dev/sda}}`
```
Comment (zdroid) [inline pages/common/badblocks.md]:
```suggestion
`sudo badblocks -o {{/path/to/file}} -w {{/dev/sda}}`
```

File: pages/common/badblocks.md [hunk 3]
```diff
@@ -14,3 +14,11 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to file:
```
Comment (zdroid) [inline pages/common/badblocks.md]:
```suggestion
- Search an unmounted disk in desctructive mode and output found blocks to a file:
```

File: pages/common/badblocks.md (hunk 1 above)
Comment (dhatch48) [reply pages/common/badblocks.md]: Yes, that is correct. I'll add the brackets around those options. I was trying to say something as to why the use of those options

**Comment by schneiderl** [comment]
> Looks good to me but I agree with @zdroid when he says that "With improved speed" is unnecessary.

Not a major problem though.

File: pages/common/badblocks.md (hunk 1 above)
Comment (mebeim) [reply pages/common/badblocks.md]: Is the notation `4k` not supported?

**Comment by dhatch48** [comment]
Why is merging still blocked if there are 2 approvals?

**Comment by mebeim** [comment]
@dhatch48 thanks for the ping, merging is blocked since @zdroid requested some changes and still did not re-submit an approval. I'll merge this since all the comments have been addressed (I also checked myself that the tool does not support size suffixes in command line arguments).

Thank you for the PR :slightly_smiling_face:

File: pages/common/badblocks.md [hunk 4]
```diff
@@ -14,3 +14,15 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk for bad blocks with a destructive write test and show verbose status:
+
+`sudo badblocks -svw {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to a file:
```
Comment (agnivade) [inline pages/common/badblocks.md]: Typo at destructive.

Also, "output" is not a verb. Let's change it to "write"

File: pages/common/badblocks.md [hunk 5]
```diff
@@ -14,3 +14,15 @@
...
+
+`sudo badblocks -o {{/path/to/file}} -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode with improved speed using 4K block size and 64K block count:
```
Comment (agnivade) [inline pages/common/badblocks.md]: Typo at destructive.

Also, "using block size of 4K and block count of 64K"

**Comment by zdroid** [comment]
@agnivade https://www.merriam-webster.com/dictionary/output

**Comment by agnivade** [comment]
TIL !

**Pull request merged**